mwdb/web/src/components/ShowObject/Actions/PullAction.js

```diff
@@ -20,15 +24,15 @@ export default function PullAction() {
             text_blob: "blob",
         }[context.object.type];
         try {
+            setPullModalDisabled(true);
             await api.pullObjectRemote(api.remote, type, context.object.id);
+            history.push(`/${match.params.type}/${match.params.id}`);
```

Comment:
seems more easy
```suggestion
            history.push(`/${type}/${context.object.id}`);
```

Reply:
Not exactly, because of the `sample` vs `file` problem (the view is `sample/` not `file/`)

Reply:
Fixed after #314